custom_components/tahoma/coordinator.py

```diff
@@ -59,9 +59,13 @@ def __init__(
         )
 
         self.data = {}
-        self.original_update_interval = update_interval
         self.client = client
         self.devices: Dict[str, Device] = {d.deviceurl: d for d in devices}
+        self.is_stateless = all(
+            device.deviceurl.startswith("rts://")
```

Comment:
Would be nice to add a protocol property within pyhoma

Reply:
An enum? Or a property in the model? 

Could we perhaps merge this one in for the mean-time? It should fix quite some users problems.

Reply:
Just a property. OK to be merged if has been tested

Reply:
I tested it and it will change the update interval for RTS installations, and keep the same one for non RTS.